about.html

```diff
@@ -0,0 +1,151 @@
+                &times;
+            </div>
+        </section>  
+        <article id="about" class="about">
+            <section class="about-sec1">
+                <div class="sub-about">
+                    
+                    <p class="para1">
+                        "Hello! Sharing world"
+                    </p>
+                    <h1 class="main-title">
+                        73RD FIFA CONGRESS MEETING
+                    </h1>
+                    
+                    <p class="para2">
+                        The FIFA Congress is the supreme 
+                        legislative body of the International 
+                        Association Football Federation 
+                        (French: Fédération Internationale de Football Association), 
+                        commonly known by the acronym FIFA. 
+                        FIFA is the international governing 
+                        body of association football, futsal and beach soccer.
+                    </p>
+                    <p class="mail">
+                        If you have any questions related to the Conference, you may email us on:
+                        <a href="#">
+                            sales@rcb.rw
+                        </a>
+                      </p>
+                </div>
+                <h2 class="about-title">
+                    FIFA CONGRESS <br> CONFERENCE 2023 <br> LOGO
+                </h2>
+                <hr class="line">
+                <p class="para3">
+                    FIFA is the international governing body of association football, futsal and beach soccer.
+                </p>
+                <div class="logo">
+                    <img src="./images/FIFA.png" alt="my-logo">
+                    <div id="gallery"></div>
+                </div>
+                <h3 class="mini-about">
+                    See the past FIFA Congress Meeting
+                </h3>
+                <div class="images-about">
+                    <div class="img-about one">                        
+                        <h3>
+                            Doha
+                            <big>2022</big>
+                        </h3>
+                    </div>
+                    <div class="img-about two">
+                        <h3>
+                            Switzeland
+                            <big>2021</big>
+                        </h3>
+                    </div>
+                </div>
+            </section>
+        </article>
+    </main>
+    <footer>
+        <section id="partners" class="foot-1">
+            <h2 class="footer-title">Partner</h2>
+            <hr class="line">
+            <ul>
+                <li><big>COCACOLA</big></li>
+                <li>ADIDAS</li>
+                <li><big>QUATAR AIRWAYS</big></li>
+                <li><small>HYUNDAI</small></li>
+                <li><i class="fab fa-airbnb"></i>Airbnb</li>
+            </ul>
+          </section>
```

Comment:
- [ ] This section doesn’t exist on the `About` page, please kindly remove it 🙏 🙏 

Your design

<img width="820" alt="Screenshot 2023-03-24 at 21 01 09" src="https://user-images.githubusercontent.com/74501523/227604813-5d7966fe-dbaa-49ff-9a71-fb44aa1382e1.png">

Original design

<img width="872" alt="Screenshot 2023-03-24 at 21 00 35" src="https://user-images.githubusercontent.com/74501523/227604978-7685c1d3-866f-458e-b95b-5a788dd87ff9.png">